tests/helpers/imports.py

```diff
@@ -1,6 +1,13 @@
-from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_8
+from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_8, _TORCHTEXT_LEGACY_DEPRECATED
 
-if _TORCH_GREATER_EQUAL_1_8:
+if _TORCHTEXT_LEGACY_DEPRECATED:
```

Comment:
Dont' we want something like

if _SHOULD_USE_TORCHTEXT_LEGACY  # checks torchtext is <0.11
    from torchtext.legacy.data import Batch, Dataset, Example, Field, Iterator, LabelField
else:
    from torchtext.data import Batch, Dataset, Example, Field, Iterator, LabelField

Reply:
@carmocca , No we don't want this. Basically, this is the current status:
- In Torchtext <0.9, these classes were in `torchtext.data` package
- In Torchtext >0.9, these classes are in `torchtext.legacy.data` package
- After Torchtext 0.11, these classes will be completely removed. 

Hope that makes it clear?

Reply:
@carmocca , No we don't want this. Basically, this is the current status:

In Torchtext <0.9, these classes were in torchtext.data package
In Torchtext >0.9, these classes are in torchtext.legacy.data package
After Torchtext 0.11, these classes will be completely removed.

Hope that makes it clear?

Reply:
> After Torchtext 0.11, these classes will be completely removed.

From both `.data` and `.legacy.data`?

Reply:
@carmocca  Yes from both